crud/common/sharding.lua

```diff
@@ -2,10 +2,19 @@ local vshard = require('vshard')
 local errors = require('errors')
 
 local BucketIDError = errors.new_class("BucketIDError", {capture_stack = false})
+local ShardingKeyError = errors.new_class("ShardingKeyError", {capture_stack = false})
 
+local dev_checks = require('crud.common.dev_checks')
 local utils = require('crud.common.utils')
 
-local sharding = {}
+local sharding_key_in_primary_index = {}
+local M = {
+    ddl_sharding_keys_cache = nil
+}
```

Comment:
M usually means "module". But here module is sharding. What does M mean here?

Reply:
M means 'module', it's a table of module with private variable for access from another modules.

Reply:
comment is not relevant now, we don't need to export caches to outside.